packages/camera/camera_android_camerax/README.md

```diff
@@ -64,11 +67,4 @@ For more information on contributing to this plugin, see [`CONTRIBUTING.md`](CON
 [6]: https://developer.android.com/media/camera/camerax/architecture#combine-use-cases
 [7]: https://developer.android.com/reference/android/hardware/camera2/CameraMetadata#INFO_SUPPORTED_HARDWARE_LEVEL_3
 [8]: https://developer.android.com/reference/android/hardware/camera2/CameraMetadata#INFO_SUPPORTED_HARDWARE_LEVEL_LIMITED
-[120462]: https://github.com/flutter/flutter/issues/120462
```

Comment:
Intentional removal?

Reply:
Yes! There aren't referenced in the README anymore.